kotlin-sqlobject/src/main/kotlin/org/jdbi/v3/sqlobject/kotlin/RegisterKotlinMapper.kt

```diff
@@ -14,20 +14,15 @@
 package org.jdbi.v3.sqlobject.kotlin
 
 import org.jdbi.v3.sqlobject.config.ConfiguringAnnotation
-import java.lang.annotation.ElementType
-import java.lang.annotation.Retention
-import java.lang.annotation.RetentionPolicy
-import java.lang.annotation.Target
 import kotlin.reflect.KClass
 
 /**
  * Registers a KotlinMapper for a specific kotlin class
  */
-@Retention(RetentionPolicy.RUNTIME)
+@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
```

Comment:
as we import other kotlin classes, could these be imports instead of fully qualified references?

Reply:
done!